src/components/ui/color-mode.tsx

```diff
@@ -0,0 +1,107 @@
+"use client";
```

Comment:
Is this required for playwright?

Reply:
This was added by a Chakra script that they recommend running which adds snippets to the codebase:
https://chakra-ui.com/docs/get-started/installation#add-snippets

I deleted some of the snippets because they didn't follow our coding conventions. I realize now I should put them in a custom folder and update the eslint rules. That way we can have unmodified snippets that are easier to update if we update Chakra. 

I don't really like this approach of using snippets because we have to then maintain them. I which Chakra would provide away to get the convenience of the snippets, but just with code built into Chakra.